packages/wrangler/src/index.ts

```diff
@@ -583,51 +572,8 @@ export function createCLIParser(argv: string[]) {
 	register.registerNamespace("logout");
 	register.registerNamespace("whoami");
 
-	/******************************************************/
-	/*               DEPRECATED COMMANDS                  */
-	/******************************************************/
-	// [DEPRECATED] build
 	wrangler.command("build", false, buildOptions, buildHandler);
```

Comment:
should this be removed as well?

Reply:
No—we've decided to keep around `wrangler build` since it's a pretty useful shortcut for `wrangler deploy --dry-run dist`